examples/plotter.py

```diff
@@ -0,0 +1,97 @@
+"""These functions compare properties of different molecules."""
+import numpy
+import warnings
+
+from fermilib.utils import (make_atom, make_atomic_ring,
+                            MolecularData, periodic_table)
+
+with warnings.catch_warnings():
+    warnings.simplefilter('ignore')
+    import pylab

```

Comment:
This is also used in the psi4 plugin. Let's change from pylab to matplotlib later, since other parts of FermiLib currently use pylab.